Other/EternalDragonDrops.cs

```diff
@@ -0,0 +1,71 @@
+//cs_include Scripts/CoreBots.cs
+
+using Skua.Core.Interfaces;
+using Skua.Core.Options;
+
+public class EternalDragonDrops
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+
+    public List<IOption> Options = new List<IOption>()
+    {
+        CoreBots.Instance.SkipOptions,
+        new Option<bool>("CanSolo", "Can solo boss?", "Can you solo Eternal Dragon?", false),
+    };
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        GetDrops();
+
+        Core.SetOptions(false);
+    }
+
+    public void GetDrops()
+    {   
+        string[] MemDrops = {
+            "Dark Embrace Of The Queen",
+            "ShadowFlame Aura",
+        };
+
+        string[] NonMemDrops = {
+            "Dark Stars",
+            "Embrace of the Queen",
+            "Enchanted Dragon's Battle Gear",
+            "Monster Queen's Locks",
+            "Monster Queen's Malicious Morph",
+            "Monster Queen's Wings",
+            "Requiescat Regina",
+            "Requiescat Regina Cane",   
+            "Requiescat Regina Hair",
+            "Requiescat Regina Locks",
+            "ShadowFlame Portal",
+            "Timestream Ravager's Sigil"
+        };
+
+        if(Core.CheckInventory(NonMemDrops)){
```

Comment:
are you using pretty formatter?
this isn't a must but insterting brackets a line before and line after the codeblock make it more readable also same system as other scripts